k8s/prod/cluster-00-overlay/ccpay/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
this should be in a fees-pay folder to be consistent with the rest

Reply:
folder has been renamed to fees-pay.